tools/gvisor-smoke-test.sh

```diff
@@ -27,7 +27,7 @@ cat > "$workdir/config" <<EOF
         "type": "gvisor",
         "vm": {
                 "count": 1,
-                "runsc_args": "--ignore-cgroups --network none"
+                "runsc_args": "--ignore-cgroups --network none --debug --debug-log=$workdir/logs/"
```

Comment:
syzkaller collects gvisor logs and parse them to detect panics and other issue. This part of syzkaller will not be tested, if a debug log is set for runsc.

Reply:
From my testing, when gVisor fails, all I got was a cut-off error message that said "`running container: creating container: cannot create gofer process: gofer: fork/exec /proc/self/exe: opera`". This PR is how I got the full logs instead.

Perhaps I can add an option similar to https://github.com/google/syzkaller/pull/5037 to change the behavior?

Reply:
@EtiennePerot syz-manager will show a vm logs if we run it with "--debug -vv 100".